addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/TestPersistentXSS1CollectOriginalParamValues.java

```diff
@@ -19,23 +19,27 @@
  */
 package org.zaproxy.zap.extension.ascanrules;
 
+import com.googlecode.concurrenttrees.radix.node.concrete.voidvalue.VoidValue;
+import com.googlecode.concurrenttrees.radixinverted.ConcurrentInvertedRadixTree;
 import org.apache.log4j.Logger;
 import org.parosproxy.paros.Constant;
 import org.parosproxy.paros.core.scanner.AbstractAppParamPlugin;
 import org.parosproxy.paros.core.scanner.Alert;
 import org.parosproxy.paros.core.scanner.Category;
 import org.parosproxy.paros.network.HttpMessage;
 
-public class TestPersistentXSSPrime extends AbstractAppParamPlugin {
+public class TestPersistentXSS1CollectOriginalParamValues extends AbstractAppParamPlugin {
 
     /** Prefix for internationalised messages used by this rule */
-    private static final String MESSAGE_PREFIX = "ascanrules.testpersistentxssprime.";
+    private static final String MESSAGE_PREFIX =
+            "ascanrules.testpersistentxss1collectoriginalparamvalues.";
 
-    private static Logger log = Logger.getLogger(TestPersistentXSSPrime.class);
+    private static Logger log =
+            Logger.getLogger(TestPersistentXSS1CollectOriginalParamValues.class);
 
     @Override
     public int getId() {
-        return 40016;
+        return 40037;
```

Comment:
Was this changed so that you can use them side-by-side?

Reply:
The original Spider and Prime scanners had 2 different Ids so I did set a different Id for each of the new ones.